test-network/network.sh

```diff
@@ -88,9 +88,9 @@ BLACKLISTED_VERSIONS="^1\.0\. ^1\.1\. ^1\.2\. ^1\.3\. ^1\.4\."
 # of go or other items could be added.
 function checkPrereqs() {
   ## Check if your have cloned the peer binaries and configuration files.
-  peer version > /dev/null 2>&1
+  which peer
 
-  if [[ $? -ne 0 || ! -d "../config" ]]; then
+  if [ "$?" -ne 0 ]; then
     echo "ERROR! Peer binary and configuration files not found.."
```

Comment:
this should be updated as well to exclude `configuration files`, right?

Reply:
right. will fix up.